tests/test_client.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+import testinfra
+import subprocess
+import os
+import time
+import json
+
+
+@pytest.fixture
+def entities():
```

Comment:
Should be centralized at some point as other tests we want to migrate here will use these.

Reply:
We already use https://pypi.org/project/pytest-variables/. That is probably the best way since it also allows overriding things.

Reply:
Given these are not things that should change, I wouldn't want to have to create them via variables every time. Perhaps a future use case will steer it that direction.

Reply:
I thought that we could include a default variables file in the repo and always load that. Doesn't look like that's currently possible, but we can probably override `pytest_configure` in `conftest.py` to always inject `defaults.json` (or whatever name) as a first value.